content/scout/integrations/environment/_index.md

```diff
@@ -32,33 +30,63 @@ assign that tag to your `production` environment. You might be running a
 different version of the same image in staging, in which case you can assign
 that version of the image to the `staging` environment.
 
+To add environments to Docker Scout, you can:
+
+- Use the `docker scout env <environment> <image>` CLI command to record images to environments manually
+- Enable a runtime integration to automatically detect images in your environments.
+
+Docker Scout supports the following runtime integrations:
+
+- [Docker Scout GitHub Action](https://github.com/marketplace/actions/docker-scout#record-an-image-deployed-to-a-stream-environment)
+- [Sysdig integration](./sysdig.md)
+
```

Comment:
Are we mentioning here about the command can be added to other CI providers like CircleCI, Jenkins, etc.?

Reply:
Not for this PR